hail/python/test/hail/experimental/test_experimental.py

```diff
@@ -439,3 +439,10 @@ def test_loop_errors(self):
                 hl.if_else(my_nd[0, 0] == 1000, my_nd, f(my_nd + 1)),
                 hl.tndarray(hl.tint32, 2), hl.nd.zeros((20, 10), hl.tfloat64))
 
+    # def test_loop_memory(self):
```

Comment:
commented because slow, right? Shall we pytest.mark.skip instead?

Reply:
Commented because slow. Note my question above: Any good way to add a test that checks that it doesn't blow up memory that isn't going to be slow?